deepfence_frontend/apps/dashboard/src/features/integrations/pages/Integrations.tsx

```diff
@@ -207,28 +248,15 @@ const Integrations = () => {
         </Breadcrumb>
       </div>
       <div className="m-4 gap-y-6 flex flex-col">
-        {IntegrationsData.map((integration) => {
+        {IntegrationsData.map((integration, index) => {
           return (
             <section key={integration.name} className="flex flex-col">
               <h2 className="uppercase text-t3 dark:text-text-input-value">
                 {integration.name}
               </h2>
-              <div className="mt-2 flex flex-wrap gap-4">
-                {integration?.types?.map((type) => {
-                  return (
-                    <DFLink to={type.path} unstyled key={type.name}>
-                      <Card className="p-3 flex flex-col shrink-0 min-w-[208px] ring-inset dark:hover:ring-bg-hover-3 dark:hover:ring-2 dark:focus:ring-bg-hover-3 dark:focus:ring-2 cursor-pointer">
-                        <div className="flex items-center gap-x-6">
-                          <div className="dark:bg-bg-grid-default rounded-full p-3 flex justify-center items-center">
-                            <span className="h-9 w-9">{type.icon}</span>
-                          </div>
-                          <Type setError={setError} type={type} />
-                        </div>
-                      </Card>
-                    </DFLink>
-                  );
-                })}
-              </div>
+              <Suspense fallback={<Skeleton count={skeletonCardCount[index]} />}>
```

Comment:
instead of `skeletonCardCount` you can do `IntegrationData[index].types.length`.